pkg/skuba/actions/cluster/status/status.go

```diff
@@ -34,7 +35,25 @@ func Status(client clientset.Interface) error {
 		return errors.Wrap(err, "could not retrieve node list")
 	}
 
-	outputFormat := "custom-columns=NAME:.metadata.name,OS-IMAGE:.status.nodeInfo.osImage,KERNEL-VERSION:.status.nodeInfo.kernelVersion,KUBELET-VERSION:.status.nodeInfo.kubeletVersion,CONTAINER-RUNTIME:.status.nodeInfo.containerRuntimeVersion,HAS-UPDATES:.metadata.annotations.caasp\\.suse\\.com/has-updates,HAS-DISRUPTIVE-UPDATES:.metadata.annotations.caasp\\.suse\\.com/has-disruptive-updates,CAASP-RELEASE-VERSION:.metadata.annotations.caasp\\.suse\\.com/caasp-release-version"
+	for _, node := range nodeList.Items {
+		for label, value := range node.Labels {
+			_ = value
```

Comment:
lint error for `for label, _ := range node.Labels {`. I added  `_ = value`.

Reply:
I don't think I understand what is going on here.... but this cant be the solution.

Reply:
Should be `for label := range nodeList.Items {` without the second value

```
$ golint pkg/skuba/actions/cluster/status/status.go 
pkg/skuba/actions/cluster/status/status.go:39:14: should omit 2nd value from range; this loop is equivalent to `for label := range ...`
```

Reply:
The outer loop iterates over `range nodeList.Items`. How would that also work on the inner loop?

Reply:
Oh, did you mean `for label := range node.Labels {`, @cmurphy? Does Go just silently discard later elements if the LHS only has one variable name provided?

Reply:
Apparently.  https://stackoverflow.com/a/49193047/65589

Ugh.  Go is horrible. :D

Reply:
Sorry yes I meant `range node.Labels`

Reply:
Thanks Calleen